### DISCUSSION TITLE
"About " content

### DISCUSSION THREAD

**Discussion by annunal** [beeware/toga#2360] [Q&A] [answered]
If I create a completely new briefcase project the scripts asks for title, version, description etc and finally produces the app  toml that is then used to build the app.

The information provided at the beginning are then used in the "About" menu item .  I tried to chaneg later on the information that are in the toml file to update the description or the verison but thiose information are ignored.  Where are them stored and where I can change them ?

Thanks
![image](https://github.com/beeware/toga/assets/10267112/692785c5-aaf1-47d0-a8a0-2f0bbc199592)

**Comment by mhsmith** [comment]
This information is updated in the same way as your app's own Python code – by running `briefcase update android` or `briefcase run android -u`.

**Reply by annunal** [reply]
Unfortunately not. If I run 

```
briefcase update android
briefcase run android

```
it is not updated in the output apk

**Comment by rmartin16** [comment]
Some data is hardcoded in to the `build` when it is created; therefore, it may be necessary to re-run `briefcase create android` (or just delete the `build` directory) when that information is updated.

Related:
- https://github.com/beeware/briefcase/issues/807
- https://github.com/beeware/briefcase/issues/472

**Reply by annunal** [reply, marked as answer]
Thanks a lot. It worked.  :)

 I have deleted the whole build directory and re-done the build. Now the About is correct.
